README.md

```diff
@@ -41,124 +40,129 @@ You can send metrics, histograms, or trace data from your application to the ser
-* Option 2 **(Deprecated)**: [Create a `WavefrontProxyClient`](#option-2-create-a-wavefrontproxyclient) to send data to a Wavefront Proxy.
-* Option 3: [Create a `WavefrontClient`](#option-3-create-a-wavefrontclient) to send data to the service directly or via proxy.
+Let's [create a `WavefrontClient`](#create-a-wavefrontclient) to send data to Operations for Applications either via Wavefront Proxy or directly over HTTP.
+
 > **Deprecated implementations**: *`WavefrontDirectClient` and `WavefrontProxyClient` are deprecated from proxy version 7.0 onwards. We recommend all new applications to use the `WavefrontClient`.*
 
-### Option 1: Create a WavefrontDirectClient
-When sending data via direct ingestion, you need to create a `WavefrontDirectClient`, and build it with the cluster URL and API token to send data directly to the service.
+### Create a WavefrontClient
 
->**Prerequisites**
-> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups, Roles, and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
-> * The URL of your cluster. This is the URL you connect to when you log in to the service, typically something like `https://<domain>.wavefront.com`.
-> * [Obtain the API token](http://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
+Use `WavefrontClientFactory` to create a `WavefrontClient` instance, which can send data directly to the service or send data using a Wavefront Proxy.
 
-#### Initialize the WavefrontDirectClient
-You initialize a `WavefrontDirectClient` by providing the access information you obtained in the Prerequisites section..
+The `WavefrontClientFactory` supports multiple client bindings. If more than one client configuration is specified, you can create a `WavefrontMultiClient` instance, which can send data to multiple services.
 
-Optionally, you can specify parameters to tune the following ingestion properties:
+#### Initialize the WavefrontClient (Wavefront Proxy/direct ingestion)
 
-* Max queue size - Internal buffer capacity of the sender. Any data in excess of this size is dropped.
-* Flush interval - Interval for flushing data from the sender directly to the service.
-* Batch size - Amount of data to send to the service in each flush interval.
+##### Prerequisites (Wavefront Proxy/direct ingestion)
 
-Together, the batch size and flush interval control the maximum theoretical throughput of the sender. You should override the defaults _only_ to set higher values.
+* Sending data via Wavefront Proxy?
+  Before your application can use a `WavefrontClient` you must [set up and start a Wavefront Proxy](https://docs.wavefront.com/proxies_installing.html).
+* Sending data via direct ingestion?
+  * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups, Roles, and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
```

Comment:
* Verify that you have the **Direct Data Ingestion** permission. For details, see [Operations for Applications Permissions in VMware Cloud Services](https://docs.wavefront.com/csp_permissions_overview.html).